icubam/www/handlers/show.py

```diff
@@ -11,6 +14,9 @@ def initialize(self, db):
 
   def get_icu_data(self):
     df = self.db.get_bedcount()
```

Comment:
Maybe add a small comment specifying that you're augmenting the original data so it's clear, otherwise nice and succint :)